pkg/specgen/specgen.go

```diff
@@ -293,7 +299,7 @@ type ContainerStorageConfig struct {
 	// These will supersede Image Volumes and VolumesFrom volumes where
 	// there are conflicts.
 	// Optional.
-	Mounts []spec.Mount `json:"mounts,omitempty"`
+	Mounts []SpecMount `json:"mounts,omitempty"`
```

Comment:
This is technically a breaking change. The raw specgen is exposed to pkg/bindings as users are likely using that to setup mounts.

Does using a `// swagger:model` comment work? i.e ref https://github.com/containers/podman/commit/0a92b399dffeacdffeeecdc7e6ad49c1ffe3881e

Reply:
let me try it now